notebooks/nvidia_haystack_medical_chatbot.ipynb

```diff
@@ -0,0 +1,322 @@
+{
```

Comment:
<p>If you have the context, it would be nice to include information about the parameters you used here. What's <code>bad</code>,<code>stop</code> ,<code>seed</code> and why we provided <code>None</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepset-ai/haystack-cookbook/pull/62/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>